mmdet3d/datasets/pipelines/data_augment_utils.py

```diff
@@ -63,19 +63,15 @@ def box_collision_test(boxes, qboxes, clockwise=True):
                             B = lines_boxes[i, k, 1]
                             C = lines_qboxes[j, box_l, 0]
                             D = lines_qboxes[j, box_l, 1]
-                            acd = (D[1] - A[1]) * (C[0] -
-                                                   A[0]) > (C[1] - A[1]) * (
-                                                       D[0] - A[0])
-                            bcd = (D[1] - B[1]) * (C[0] -
-                                                   B[0]) > (C[1] - B[1]) * (
-                                                       D[0] - B[0])
+                            acd = (D[1] - A[1]) * (C[0] - A[0]) > (
+                                C[1] - A[1]) * (D[0] - A[0]) + 1e-5
```

Comment:
Why do we need `1e-5` here?

Reply:
In different environments, the input of this function is different, resulting in different output. by adding 1e-5, we can get the same output in different environments.